cli-commands/epoch/epoch.js

```diff
@@ -157,8 +153,15 @@ async function run(option) {
     } else {
       if (!running) {
         print('===== Starting epoch =====');
-
-        sdkInstallProcess = spawn('docker-compose', ['up', '-d'], {});
+        dockerProcess = spawn('docker-compose', ['up', '-d']);
+
+        dockerProcess.stdout.on('data', (data) => {
+          print(data.toString())
```

Comment:
This will print the whole data to the console? Do we want that or just a message that the epoch is started?

Reply:
Done